index.js

```diff
@@ -154,8 +180,14 @@ module.exports = function configParser(
         // Flatten structures
         .then((parsedDoc) => {
             warnMessages = warnMessages.concat(validateTemplateVersion(parsedDoc, templateFactory));
+            warnMessages = warnMessages.concat(validateOrderAndTemplate(parsedDoc));
```

Comment:
is this separate fn needed ? can't `phaseFlatten` handle it ? it's anyways doing validation for step existence ?

Reply:
updated